src/pudl/analysis/ferc1_eia.py

```diff
@@ -1012,3 +1013,691 @@ def add_null_overrides(connects_ferc1_eia):
     ] = [np.nan] * len(eia_cols_to_null) + ["overridden"]
 
     return connects_ferc1_eia
+
+
+#######################################################################################
+# Create Training Data / Override Tools
+#######################################################################################
+# """
+# Create output spreadsheets used for overriding and checking FERC-EIA record matches.
+
+# The connect_ferc1_to_eia.py module uses machine learning to link records from FERC
+# with records from EIA. While this process is waaay better more efficient and logical
+# than a human, it requires a set of hand-compiled training data in order to do it's job.
+# On top of that, it's not always capable of mapping some of the trickier records that
+# humans might be able to make sense of based on past or surrounding records.
+
+# This module creates an output spreadsheet, based on a certain utility, that makes the
+# matching and machine-matched human validation process much easier. It also contains
+# functions that will read those new/updated/validated matches from the spreadsheet,
+# validate them, and incorporate them into the existing training data.
+
+# """
```

Comment:
I honestly think the two steps (being 1. make the spreadsheets for manual mapping & 2. integrate those spreadsheets into the training data) could be a little more separated. It may be easier to have two modules and/or two notebooks. maybe even just the two notebooks seem better.

Reply:
Ugh, I agree, I just wasn't sure how much time it was worth spending on this. I think we should have a convo about what this process will look like moving forward before refactoring.

Reply:
I could also make step 1 into a script?

Reply:
I could also make the validation step into a script...

Reply:
Verdict: not making them into scripts for time. Just made two separate notebooks